prusti-contracts/src/lib.rs

```diff
@@ -35,6 +35,12 @@ mod private {
     /// A macro for defining a predicate using prusti expression syntax instead
     /// of just Rust expressions.
     pub use prusti_contracts_impl::predicate;
+
+    /// A macro for specifying specs to export
+    pub use prusti_contracts_impl::export_spec;
+
+    /// A macro for specifying specs to import
```

Comment:
Please make the comments the same in the two versions (here and L82). We should eventually get rid of the needless duplication but still…